static/app/utils/replays/replayReader.tsx

```diff
@@ -589,11 +586,18 @@ export default class ReplayReader {
     this._sortedSpanFrames.filter((frame): frame is MemoryFrame => frame.op === 'memory')
   );
 
+  getCustomFrames = memoize(() =>
+    this._sortedBreadcrumbFrames.filter(
+      frame => !BreadcrumbCategories.includes(frame.category)
```

Comment:
This `BreadcrumbCategories` list looks like it also includes 'console'

Reply:
Custom frames is anything in the sorted breadcrumbs frames list that isn't in `BreadcrumbCategories`. So, console logs won't be in the breadcrumbs list